templates/taxi/car_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update, Create" }} new car model</h1>
```

Comment:
The `yesno` filter is incorrectly applied to `object`. This filter is meant for boolean values, not objects. You should determine whether the form is for updating or creating a new car model through logic in the view or by checking if the object exists, and then pass the appropriate context variable to the template.